tools/tests/test_nd_freeze

```diff
@@ -127,8 +127,7 @@ assert_line_in_file "sources.list" "# deb http://deb.debian.org/debian jessie-up
 assert_line_in_file "sources.list.orig.disabled" "deb http://deb.debian.org/debian jessie main"
 assert_line_in_file "sources.list.orig.disabled" "deb http://security.debian.org" "partial_match"
 assert_line_in_file "sources.list.orig.disabled" "deb http://deb.debian.org/debian jessie-updates main"
-assert_line_in_file "stdout" "INFO: Will DOWNGRADE 'gcc-4.9-base:amd64=4.9.2-10+deb8u1' to version '4.9.2-10'" "partial_match"
-assert_line_in_file "stdout" "INFO: Will DOWNGRADE 'ncurses-base=5.9+20140913-1+deb8u3' to version '5.9+20140913-1'" "partial_match"
+assert_line_in_file "stdout" "INFO: DOWNGRADING: base-files=8+deb8u9 gcc-4.9-base:amd64=4.9.2-10" "partial_match"
```

Comment:
Sweet!  Greedy me would love to see also smth like `assert_not_line_in_file` for at least one of them while running in non-debug mode, and make one run in debug mode and assess that some expected debug line is in there